examples/qlever_e2e/gantry.def.yml

```diff
@@ -4,7 +4,7 @@ services:
   qlever:
     image: niklas88/qlever
     volumes:
-      - ./index:/index
+      - "{{TempDir \"index\"}}:/index"
```

Comment:
Since it is very useful to keep the index e.g. to then use `--ignore-steps` to only launch the server, I would prefer not to put the index in a temporary directory. Would the current version already support using a variable filled from the env file here? Then you could demonstrate the `TempDir` for `"input"` and using the env file with a default of `"./index"` preserving the index.

Reply:
Using a Path provided by the env is the second step. Will be a second pull request so that this temporary-path feature can be used soon.